src/main/java/com/java_essential/safonov/homework/lesson7/Price/Main.java

```diff
@@ -0,0 +1,50 @@
+package com.java_essential.safonov.homework.lesson7.Price;
+/*
+Требуется: Описать класс с именем Price, содержащую следующие поля:
+ название товара;
+ название магазина, в котором продается товар;
+ стоимость товара в гривнах.
+Написать программу, выполняющую следующие действия:
+ввод с клавиатуры данных в массив, состоящий из двух элементов типа Price
+(записи должны быть упорядочены в алфавитном порядке по названиям магазинов);
+вывод на экран информации о товарах, продающихся в магазине, название
+которого введено с клавиатуры (если такого магазина нет, вывести исключение).
+
+ */
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class Main {
+    public static void main(String[] args) throws Exception {
```

Comment:
Очень плохая практика выбрасывать исключение из мейн метода,
обработайте согласно лучших практик и используйте логирование 
```suggestion
    public static void main(String[] args) {
```